bundles/framework/personaldata/Flyout.js

```diff
@@ -141,7 +158,16 @@ Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.Flyout',
                 var panel = Oskari.clazz.create('Oskari.userinterface.component.TabPanel');
                 panel.setTitle(tab.getTitle());
                 panel.setId(tabId);
-                tab.addTabContent(panel.getContainer());
+
+                if (tab.hasOwnProperty('getJsx')) {
+                    panel.setContent(ReactDOMServer.renderToString(tab.getJsx(), document.createElement('div')));
```

Comment:
I'm not sure what using ReactDOMServer means here, but could we do it like this:
```suggestion
                    const container= document.createElement('div');
                    panel.setContent(container);
                    ReactDOM.render(tab.getJsx(), container);
```

Reply:
Changed like suggested.